### ISSUE TITLE
Return with a range of free ports

### ISSUE THREAD

**Issue by ehartford** [indexzero/node-portfinder#3]
I want to spin up two servers.  So I run two getPorts.  However, the servers step on each other.

```javascript
portfinder = require("portfinder");

portfinder.getPort(function(err1,port1){
  portfinder.getPort(function(err2, port2){
    console.log("Port1 = " + port1 + ", Port2 = " + port2);
  });
});
```

outputs:
"Port1 = 8000, Port2 = 8000"

**Comment by charlierudolph** [comment]
@indexzero can you release a new version now that #11 is in? I can add some documentation if needed.

**Comment by indexzero** [comment]
This is released in `0.4.0`
